.github/workflows/unit-test.yml

```diff
@@ -8,11 +8,11 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: ['14', '16.8']
+        node-version: ['14', '16']
         include:
           - node-version: '14'
             node-name: 'LTS'
-          - node-version: '16.8'
+          - node-version: '16'
```

Comment:
Did Node.js fix the issue that caused jest crashing?

Reply:
Yes. https://github.com/nodejs/node/issues/40030